server/coin-scraper.js

```diff
@@ -9,25 +9,30 @@ module.exports = class CoinScraper {
     this.previousItemsMap = {};
   }
 
-  fetchMostRecentListedItems() {
-    const requestOptions = {
-      method: 'GET',
-      uri: 'https://pro-api.coinmarketcap.com/v1/cryptocurrency/listings/latest',
-      qs: {
-        'start': '1',
-        'limit': '200',
-        'sort': 'date_added',
-        'sort_dir': 'asc',
-        'aux': 'date_added'
-      },
-      headers: {
-        'X-CMC_PRO_API_KEY': CMC_API_KEY
-      },
-      json: true,
-      gzip: true
-    };
-
-    return rp(requestOptions);
+  async fetchMostRecentListedItems() {
+    try {
+      const requestOptions = {
+        method: 'GET',
+        url: 'https://pro-api.coinmarketcap.com/v1/cryptocurrency/listings/latest',
+        qs: {
+          'start': '1',
+          'limit': '500',
```

Comment:
@Shapokhan limit 200 hi rakhni hai cuz it costs 1 call credit for every 200 items fetched via this call. New arrivals tou thoray se hi hotay hain between any two calls, so even 200 is more than enough items to fetch cuz these items are already sorted on date_added.

Reply:
Changed to 200.